docs/data/charts/radar/radar.md

```diff
@@ -59,12 +59,6 @@ The radar chart displays a grid behind the series that can be configured with:
 
 {{"demo": "DemoRadar.js" }}
 
-## Interaction 🚧
-
-### Axis click 🚧
-
-### Item click 🚧
-
```

Comment:
I removed those sections, because for me it's not necessary for a stable release